src/stirling/source_connectors/socket_tracer/socket_trace_connector.cc

```diff
@@ -170,13 +170,16 @@ constexpr size_t kMaxHTTPHeadersBytes = 8192;
 // Protobuf printer will limit strings to this length.
 constexpr size_t kMaxPBStringLen = 64;
 
+constexpr char openssl_mismatched_fds_metric[] = "openssl_trace_mismatched_fds";
+constexpr char openssl_mismatched_fds_help[] =
+    "Count of the times a syscall's fd was mismatched when detecting fds from an active user space "
+    "call";
+
 SocketTraceConnector::SocketTraceConnector(std::string_view source_name)
     : SourceConnector(source_name, kTables),
       conn_stats_(&conn_trackers_mgr_),
-      openssl_trace_mismatched_fds_counter_(BuildCounter(
-          "openssl_trace_mismatched_fds",
-          "Count of the times a syscall's fd was mismatched when detecting fds from an "
-          "active user space call")),
+      openssl_trace_mismatched_fds_counter_(
+          BuildCounter(openssl_mismatched_fds_metric, openssl_mismatched_fds_help)),
```

Comment:
I think we should call `BuildCounterFamily` here, and then call `...family_.Add(...).Increment()` on the class member below

Reply:
Updated.